inkscape-1.3.2_2023-11-25_091e20ef0f/src/ui/widget/scalar.cpp:123:32: error: call of overloaded 'pow(double, guint)' is ambiguous
  123 |     double digits = (double)pow(10.0,static_cast<SpinButton*>(_widget)->get_digits());
      |                             ~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

--- inkscape-1.3.2_2023-11-25_091e20ef0f/src/ui/widget/scalar.cpp.old	2024-01-28 15:15:29.039315299 -0500
+++ inkscape-1.3.2_2023-11-25_091e20ef0f/src/ui/widget/scalar.cpp	2024-01-28 15:15:40.199256897 -0500
@@ -120,7 +120,7 @@
 Scalar::setNoLeadingZerosOutput()
 {
     g_assert(_widget != nullptr);
-    double digits = (double)pow(10.0,static_cast<SpinButton*>(_widget)->get_digits());
+    double digits = (double)std::pow(10.0,static_cast<SpinButton*>(_widget)->get_digits());
     double val = std::round(static_cast<SpinButton*>(_widget)->get_value() * digits) / digits;
     static_cast<SpinButton*>(_widget)->set_text(Glib::ustring::format(val).c_str());
     return true;
